docs/release/3.4/release-3-4-12.md

```diff
@@ -0,0 +1,143 @@
+OSG Software Release 3.4.12
+===========================
+
+**Release Date**: 2018-05-10
+
+Summary of changes
+------------------
+
+!!! note
+The repositories referenced by osg-release will be retired on May 15th. Update osg-release before that date.
+
+``` console
+root@host # yum update osg-ca-certs-updater osg-ca-scripts osg-release osg-test rsv
+```
```

Comment:
This will install the packages if they don't have them installed. Maybe make this multi-step: 1. find packages you have installed with `rpm -q <pkg list>` then 2. update the packages you found with `yum update <installed pkg list>`

Reply:
No, yum will report to you that some packages are not installed and not update them. This command is safe as is. (It tried it out on a fermicloud VM.)